src/components/photoGallery/PhotoGallery.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import "./PhotoGallery.scss"
+
+export const PhotoGallery: React.FunctionComponent = () => {
+
+  const [imageUrls, setImageUrls] = useState<string[]>();
+  const img_urls: string[] = [];
```

Comment:
`camelCase` for JavaScript and TypeScript variables!
```suggestion
  const imgUrls: string[] = [];
```

Although I'm confused as to why you have both `imageUrls` and `img_urls`. What are they used for?

Reply:
Please remove `img_urls` as I don't think it's needed!
```suggestion
```